book/routing.rst

```diff
@@ -1429,7 +1441,10 @@ Generating URLs with Query Strings
 The ``generate`` method takes an array of wildcard values to generate the URI.
 But if you pass extra ones, they will be added to the URI as a query string::
 
-    $this->get('router')->generate('blog', array('page' => 2, 'category' => 'Symfony'));
+    $this->get('router')->generate('blog', array(
+        'page' => 2,
+        'category' => 'Symfony'
```

Comment:
Crap, missed the comma, losing my sharpness I guess :D

Yea, usually docs have the trailing comma and as contribution docs has a section about having trailing comma, in my opinion it's a good idea to follow that rule in docs also.